arm-software/linux/README.md

```diff
@@ -303,3 +303,11 @@ $ armclang -fopenmp -o example example.o `pkg-config armpl-dynamic-lp64-omp --li
 $ chrpath -l ./example
 ./example: RUNPATH=$HOME/armpl/armpl_24.10_flang-new/lib/pkgconfig/../../lib:$HOME/atfl/lib/clang/20/lib/aarch64-unknown-linux-gnu:$HOME/atfl/bin/../lib/aarch64-unknown-linux-gnu
 ```
+
+## Providing feedback and reporting issues
+
+Please raise an issue via [Github issues](https://github.com/arm/arm-toolchain/issues).
+
+## Contributions and Pull Requests
+
+Please see the [Contribution Guide](docs/contributing.md) for details.
```

Comment:
Does this page exist?

Reply:
Ah, good spot: this come from the version before the fix https://github.com/arm/arm-toolchain/pull/302 - let me correct.

Reply:
Fixed.

Reply:
It refers to https://github.com/arm/arm-toolchain/blob/arm-software/CONTRIBUTING.md so it seems OK.